src/execute.rs

```diff
@@ -181,7 +182,7 @@ impl CommandsRunner {
         let child = process::Command::new(command)
             .args(args)
             .envs(env)
-            .current_dir(self.script_directory.path())
+            .current_dir(env::current_dir().unwrap())
```

Comment:
I think the default may be this already, you may be able to avoid calling `.current_dir(...)` altogether. Can you give it a shot that way?

Reply:
Derp, you're right. Removed it!